corehq/messaging/scheduling/templates/scheduling/partials/conditional_alert_save.html

```diff
@@ -1,7 +1,27 @@
 {% load i18n %}
 <div class="form-actions">
+
+  <div class="modal fade" id="conditional-alert-warning-modal" role="dialog">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal">&times;</button>
+          <h4 class="modal-title">Conditional Alert</h4>
+        </div>
+        <div class="modal-body">
+          <p>This update may alert <span id="case-count"></span> cases. Are you sure you want to proceed?</p>
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-default" data-dismiss="modal" id="conditional-alert-yes-btn">Yes</button>
```

Comment:
Use `btn-primary` for this, since it's what we expect the user to click on. Button guidelines are here: https://www.commcarehq.org/styleguide/molecules/#molecules-buttons